### ISSUE TITLE
[Feature Request] Poke stop info in the log file

### ISSUE THREAD

**Issue by avexus** [PokemonGoF/PokemonGo-Bot#5388]
When we visit Poke Stops, can we have their info written into the log file?
Like Name and Location, so that when we want to make a path, we have all the info handy.

**Comment by Gobberwart** [comment]
Name already gets written. What other info would you like? GPS coordinates?

**Comment by avexus** [comment]
@Gobberwart Yes, GPS coordinates pls. Name doesn't seem to much for path.json file.
Thank you.

**Comment by prusswan** [comment]
Yes, this will be really useful.
